prisma/migrations/20240804092018_creating_movie_model/migration.sql

```diff
@@ -0,0 +1,12 @@
+-- CreateTable
+CREATE TABLE "Movie" (
+    "id" SERIAL NOT NULL,
+    "title" TEXT NOT NULL,
+    "overview" TEXT NOT NULL,
+    "genres" JSONB NOT NULL,
```

Comment:
Perhaps a separate table for genres is what I would expect.

Reply:
I initially started with a single table and, partway through the project, considered switching to a separate table for genres. However, since this was my first time using the seeding method, I chose to stick with the original approach to avoid complications. I plan to learn and implement the separate table approach as well.